packages/sanity/src/ui-components/dialog/Dialog.tsx

```diff
@@ -9,6 +9,7 @@ import {
 } from '@sanity/ui'
 import type * as React from 'react'
 import {type ComponentProps, forwardRef} from 'react'
+//eslint-disable-next-line @sanity/i18n/no-i18next-import -- avoiding a circular import
 import {useTranslation} from 'react-i18next'
```

Comment:
It seems problematic to import from `sanity` here, and traversing up and down the file tree to get to `core` also seems problematic at best.